src/Raven.Server/Rachis/RachisConsensus.cs

```diff
@@ -1376,6 +1385,13 @@ public unsafe bool RemoveEntryFromRaftLog(long index)
             return lastIndex;
         }
 
+        private void ThrowTooLargeRaftCommand(BlittableJsonReaderObject cmd)
+        {
+            var type = RachisLogHistory.GetTypeFromCommand(cmd);
+            throw new ArgumentOutOfRangeException(
+                $"The command '{type}' size of {new Size(cmd.Size, SizeUnit.Bytes)} exceed the max allowed size.");
```

Comment:
Need to add the actually allowed size and point to the right configuration entry